src/nft-price-checker.go

```diff
@@ -214,11 +220,11 @@ func writeFile(file string, str string) {
 
 func main() {
 
-	outputType := ""
-	if len(os.Args) > 1 {
-		outputType = os.Args[1]
+	if len(os.Args) < 2 {
+		log.Fatalln("Expecting at least two Args, with os.Args[1] either equal to \"web2\" or \"web3\"")
```

Comment:
The message could be confusing, actually we are expecting 1 argument from the user since Args[0] is the script itself. So I suggest:
```suggestion
		log.Fatalln("Expecting one argument \"web2\" or \"web3\"")
```